com.unity.sg2/Editor/GraphUI/DataModel/SGNodeModel.cs

```diff
@@ -214,9 +218,17 @@ public void SetSearcherPreviewRegistryKey(RegistryKey key)
 
         public void ChangeNodeFunction(string newFunctionName)
         {
```

Comment:
The future goal is to move node function selection to a portless port, right?

Reply:
This is on my plate.

Reply:
Ah- thanks for sharing.

Yea-- it doesn't really matter how it's exposed-- but on our end, it's a static/field port/whatever-- an _input_ that retopologizes. Which I previously was hoping/expecting that portless ports could be a way to universally represent all inputs (if they had enough options to indicate how/where they should show up).

But it looks like it's going to be a different entry point for GTF.